codechecker/CodeChecker

```diff
@@ -69,6 +69,15 @@ def main():
 
     original_env_file = os.path.join(tmp_dir, 'original_env.pickle')
 
+    def _remove_tmp():
+        # Remove temporary directory.
+        try:
+            shutil.rmtree(tmp_dir)
+        except Exception as ex:
```

Comment:
It would be a nice simplification if we would handle `OSError` exceptions separately like `No such file or directory` in that case we do not have to check if the directory exists at line 104.